app/main/forms/suppliers.py

```diff
@@ -83,8 +83,8 @@ def validate(self):
 # "Add" rather than "Edit" because this information can only be set once by a supplier
 class AddCompanyRegisteredNameForm(FlaskForm):
     registered_company_name = DMStripWhitespaceStringField('Registered company name', validators=[
-        InputRequired(message="You must provide a registered company name."),
-        Length(max=255, message="You must provide a registered company name under 256 characters.")
+        InputRequired(message="Enter a registered company name."),
```

Comment:
Change 'a' to 'your':

Enter your registered company name
Enter your registered company name under 256 characters